EzPC/EzPC/main.ml

```diff
@@ -83,7 +84,7 @@ let tc_and_codegen (p:program) (file:string) :unit =
              else p
            in
            let p =
-             if Config.get_codegen () = CPP || Config.get_codegen () = CPPRING 
+             if Config.get_codegen () = CPP || Config.get_codegen () = CPPRING || Config.get_codegen () = CPPFLOAT 

```

Comment:
Why is CPPFLOAT different from CPP? Can they be merged into one?

Reply:
Codegencppfloat works for both cpp and cppfloat, modified main.ml to use Codegencppfloat for both.
Naming convention is kept separate for CPP and CPPFLOAT keeping in mind Athos, as it will require cpp separately to test fixed point and floating point backends.